content/en/cloud_cost_management/reporting/explorer.md

```diff
@@ -46,6 +46,30 @@ Use the Cost Explorer to:
 
 **Note:** When querying costs across multiple providers, resource-level tags are not available. To access resource-specific tags, filter to a single provider in your query.
 
+## Cost Change Summary Side Panel 
+
+The Cost Change Summary panel highlights what and who may be driving cost changes for the current period vs the prior period.
+
+   {{< img src="cloud_cost/reporting/cost-change-sidepanel.png" alt="The Cost Change Summary panel highlights what and who may be driving cost changes for the current period vs the prior period." style="width:100%;" >}}
```

Comment:
what do you think of using this image instead?
<img width="1732" height="1090" alt="cost-change-side-panel" src="https://github.com/user-attachments/assets/5c3e225e-649a-4d47-b00b-e01ab168cf0d" />